File: Components/WinForms/CS/RichEditMainDemo/Modules/Tables.cs
using System;

namespace DevExpress.XtraRichEdit.Demos {
    public partial class TablesModule : DevExpress.XtraRichEdit.Demos.TutorialControl {
        public TablesModule() {
            InitializeComponent();
            OpenXmlLoadHelper.Load("ActiveCustomers.docx", richEditControl);
            new RichEditDemoExceptionsHandler(richEditControl).Install();
        }

        public override RichEditControl PrintingRichEditControl { get { return richEditControl; } }

        protected override void DoShow() {
            base.DoShow();
            richEditControl.Focus();
        }
    }
}

File: Components/WinForms/VB/RichEditMainDemo/Modules/Tables.vb
Imports Microsoft.VisualBasic
Imports System

Namespace DevExpress.XtraRichEdit.Demos
    Partial Public Class TablesModule
        Inherits DevExpress.XtraRichEdit.Demos.TutorialControl
        Public Sub New()
            InitializeComponent()
            OpenXmlLoadHelper.Load("ActiveCustomers.docx", richEditControl)
            CType(New RichEditDemoExceptionsHandler(richEditControl), RichEditDemoExceptionsHandler).Install()
        End Sub

        Public Overrides ReadOnly Property PrintingRichEditControl() As RichEditControl
            Get
                Return richEditControl
            End Get
        End Property

        Protected Overrides Sub DoShow()
            MyBase.DoShow()
            richEditControl.Focus()
        End Sub
    End Class
End Namespace
